modules/bloxone_infra_host_azure/README.md

```diff
@@ -0,0 +1,96 @@
+<!-- BEGIN_TF_DOCS -->
+# Terraform Module to create BloxOne Host in Azure
+
+This module will provision an Azure virtual machine that uses a BloxOne image.
+The virtual machine will be configured to join a BloxOne Cloud Services Platform (CSP) with the provided join token.
+If a join token is not provided, a new one will be created.
+
+The BloxOne Host created in the CSP is created automatically, and cannot be managed through terraform.
+A `bloxone_infra_hosts` data source is provided to retrieve the host information from the CSP.
+The data source will use the `tags` variable to filter the hosts.
+A `tf_module_host_id` tag will be added to the tags variable so that the data source can uniquely find the host.
+
+This module will also create a BloxOne Infra Service for each service type provided in the `services` variable.
+The service will be named `<service_type>_<host_display_name>`.
+
+## Example Usage
+
+```hcl
+module "bloxone_infra_host_azure" {
+  source = "github.com/infobloxopen/terraform-provider-bloxone/modules/bloxone_infra_host_azure"
```

Comment:
https://developer.hashicorp.com/terraform/language/modules/sources#modules-in-package-sub-directories

```suggestion
  source = "github.com/infobloxopen/terraform-provider-bloxone//modules/bloxone_infra_host_azure"
```